docs/collecting-data/collecting-from-own-applications/javascript-trackers/browser-tracker/browser-tracker-v3-reference/advanced-usage/using-an-id-service/index.md

```diff
@@ -0,0 +1,154 @@
+---
+title: "Using an ID service"
+sidebar_position: 3000
+---
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Block5966 from "@site/docs/reusable/javascript-tracker-release-badge-v3/_index.md"
+
+<Block5966/>
+```
+
+## What is an ID service ?
+
+An _ID service_, as we chose to call it, is a process that allows for generating a unique browser identifier and enhancing the Snowplow tracking capabilities for the Apple Safari browser.
```

Comment:
> We up front should emphasise this is for enabling First-Party Analytics to not look like this is subverting the privacy protections too badly.

Yes I had this in mind while choosing the correct phrasing and ended up with `enhancing the Snowplow tracking capabilities` 

> Should this specify ITP rather than "Apple Safari", given any browser on iOS for instance is impacted, not just Safari?

Yeap sure, good call :) Changed it to:

`and enhancing the Snowplow tracking capabilities in environments where the Intelligent Tracking Prevention (ITP) feature is enabled such as on iOS browsers (Safari, Chrome, Firefox) and desktop Safari.`

Reply:
> We up front should emphasise this is for enabling First-Party Analytics to not look like this is subverting the privacy protections too badly.

Yes I had this in mind while choosing the correct phrasing and ended up with `enhancing the Snowplow tracking capabilities` 

> Should this specify ITP rather than "Apple Safari", given any browser on iOS for instance is impacted, not just Safari?

Yeap sure, good call :) Changed it to:

`and enhancing the Snowplow tracking capabilities in environments where the Intelligent Tracking Prevention (ITP) feature is enabled such as on iOS browsers (Safari, Chrome, Firefox) and desktop Safari.`